syntax = "proto3";

package common;

option go_package = "github.com/ahmdfdhilah/arga-workforce-service/proto/common";

message Pagination {
  int32 page = 1;
  int32 limit = 2;
}

message PaginationInfo {
  int32 page = 1;
  int32 limit = 2;
  int32 total_items = 3;
  int32 total_pages = 4;
}

message StatusResponse {
  bool success = 1;
  string message = 2;
}
